openfisca_core/taxbenefitsystems.py

```diff
@@ -244,8 +244,23 @@ def get_column(self, column_name, check_existence = False):
     def update_column(self, column_name, new_column):
         self.column_by_name[column_name] = new_column
 
+    def neutralize_variable(self, variable_name):
+        """
+        Neutralizes an OpenFisca variable existing in the tax and benefit system.
+
+        A neutralized variable always returns its default value when being computed.
+
+        Trying to set inputs for a neutralized variable has no effect.
+        """
+        self.update_column(variable_name, neutralize_column(self.get_column(variable_name)))
+
     def neutralize_column(self, column_name):
-        self.update_column(column_name, neutralize_column(self.get_column(column_name)))
+        print(
+            u"Warning: the neutralize_column method has been renamed to neutralize_variable. "
```

Comment:
I can help setting those logging handlers.

Handlers are set at the application / script (OpenFisca-Web-API) level, not the library (OpenFisca-Core) level. But the library can provide default settings.